stable_baselines/trpo_mpi/trpo_mpi.py

```diff
@@ -424,9 +427,6 @@ def fisher_vector_product(vec):
                                     grad = self.allmean(self.compute_vflossandgrad(mbob, mbob, mbret, sess=self.sess))
                                     self.vfadam.update(grad, self.vf_stepsize)
 
-                    for (loss_name, loss_val) in zip(self.loss_names, mean_losses):
```

Comment:
mean_losses out of the scope where it is not initialized. In case if policy is not updated the Exception is thrown and the program closes